app/src/pages/inside/manualLaunchesPage/manualLaunchExecutionPage/mixins.scss

```
@@ -0,0 +1,23 @@
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+@mixin file-dropzone-active {
+  border-color: var(--rp-ui-base-topaz-focused) !important;
+  background-color: rgba(0, 176, 209, 0.06) !important;
+}
+
+@mixin file-droparea-wrapper($dropzone-class) {
+  :global(.rp-ui-fp__dropzone--active),
```

Stylelint (17.11.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)